src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/XHMMTargetLikelihoodsCalculator.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.random.RandomGenerator;
+import org.broadinstitute.hellbender.tools.exome.Target;
+
+/**
+ * Implements the TargetikelihoodCalculator interface for the original XHMM-based germline model.
+ *
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public class XHMMTargetLikelihoodsCalculator implements  TargetLikelihoodCalculator {
```

Comment:
extra white space after `implements`.  also, should it be `XHMMTargetLikelihoodsCalculator` or `XHMMTargetLikelihoodCalculator`?